sqlglot/generator.py

```diff
@@ -903,7 +903,9 @@ def unique_sql(self, expression):
         return f"UNIQUE ({columns})"
 
     def if_sql(self, expression):
-        return self.case_sql(exp.Case(ifs=[expression], default=expression.args.get("false")))
+        case = exp.Case(ifs=[], default=expression.args.get("false"))
```

Comment:
this is tricky, i see you’re doing jt because the init mutate

Reply:
i don’t know if we should do it like this. may be cleaner to do a copy? i need to think

Reply:
copy is easier and also works just figured you didn't want the overhead

Reply:
i don't like overhead but, alas